test/support/builders/metadata/base.js

```diff
@@ -111,6 +112,9 @@ module.exports = class BaseMetadataBuilder {
   }
 
   ino (ino /*: number */) /*: this */ {
+    if (process.platform === 'win32') {
+      this.doc.fileid = statsBuilder.fileIdFromNumber(ino)
```

Comment:
Should we remove the `fileid()` method?

Reply:
I believe I still had to use it somewhere, but I may be wrong. Will check. Thanks!

Reply:
I was wrong. Good catch!